optashift-employee-rostering-server/src/main/java/org/optaplanner/openshift/employeerostering/server/roster/RosterRestServiceImpl.java

```diff
@@ -67,6 +77,11 @@
     @Inject
     private ShiftRestService shiftRestService;
 
+    private Map<Integer, Map<Shift, IndictmentView>> tenantIdToShiftIndictmentMap = new ConcurrentHashMap<>();
+    private Map<Integer, Map<Employee, IndictmentView>> tenantIdToEmployeeIndictmentMap = new ConcurrentHashMap<>();
+    private Map<Integer, OffsetDateTime> tenantIdToShiftIndictmentMapUpdateTime = new ConcurrentHashMap<>();
+    private Map<Integer, OffsetDateTime> tenantIdToEmployeeIndictmentMapUpdateTime = new ConcurrentHashMap<>();
```

Comment:
If they need to exist, they need to existing in the SolverManager, not in the RosterRestService.

Reply:
I'll work through this. Fix the other comments in the PR and let me know - I 'll work on top of it to design RosterRestServiceImpl and WannabeSolverManager to how I see it. Also rebase this PR to master if you haven't already.